samples/routing/message-forwarding/sample.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Message forwarding
+summary: Forwarding a copy of every message to another destination after it is processed
+component: Core
+reviewed: 2020-07-22
+---
+
+In complex upgrade scenarios it can be useful to forward a copy of processed messages to another destination. This allows an old version of an endpoint to run side-by-side with a new version, both processing the message until the old version can be fully retired.
```

Comment:
Do we need to mention (here or somewhere else) that this can be achieved out of the box if they are using ServiceControl (using the forwarding queue setting)? Or is this conceptually for a different purpose altogether?